app.ts

```diff
@@ -13,16 +15,26 @@ const app = express();
 
 // Rate limit middleware
 app.use(applicationRateLimiter); // rate-limit applied to all the routes by default
-
 var constantPath = './src/modules/';
-var routes = {};
 
+const APIrouter = express.Router();
+var Analytics : IAnalytics = null;
+if(process.env.NODE_ENV !== "test"){
```

Comment:
Can we change this from a node_env to be a feature flag instead so we can turn this off with just a rebuild 👍

Reply:
@ageddesi - NODE_ENV is set to Test when you run an npm test - so this environment setting is automatically added by the testing process.

Reply:
Have added ANALYTICS_ENABLED - which is read from env, or defaults to false